crates/rpc/rpc/src/validation.rs

```diff
@@ -44,6 +49,18 @@ pub struct ValidationApi<Provider, E: ConfigureEvm> {
     inner: Arc<ValidationApiInner<Provider, E>>,
 }
 
+fn hash_disallow_list(disallow: &HashSet<Address>) -> String {
+    let mut sorted: Vec<_> = disallow.iter().collect();
+    sorted.sort(); // sort for deterministic hashing
+
+    let mut hasher = Sha256::new();
+    for addr in sorted {
+        hasher.update(addr.as_slice()); // Use as_slice() for Address bytes
+    }
+
+    format!("{:x}", hasher.finalize()) // lowercase hex string
```

Comment:
those comments we can remove here

should this be with 0x prefix?

Reply:
Without 0x is fine, it matches rbuilder implementation